scripts/ci/run_notebook.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+notebook="$1"
+# papermill will replace parameters on some notebooks to make them run faster in CI
+papermill --execution-timeout=600 \
+  --parameters_file "./.buildkite/notebook-parameters.yml" --log-output \
+  "$notebook" "$notebook"
+
+notebook_name="$(basename "$notebook")"
+echo "::set-output name=notebook_name::$notebook_name"
```

Comment:
I used "environment" rather than "output" in my PR - this looks better, will update mine 👍